service/src/main/resources/application-local.yaml

```diff
@@ -110,10 +113,13 @@ fpl:
     username: 'fpl-system-update@mailnesia.com'
     password: 'Password12'
   ctsc_inbox: 'FamilyPublicLaw+ctsc@gmail.com'
-  ctsc_team_lead_inbox: 'FamilyPublicLaw+ctsc-team-lead@gmail.com'
+  ctsc_team_lead_inbox: 'FamilyPlublicLaw+ctsc-team-lead@gmail.com'
```

Comment:
small typo!
```suggestion
  ctsc_team_lead_inbox: 'FamilyPublicLaw+ctsc-team-lead@gmail.com'
```

Reply:
thanks! didn't actually mean to change that!